docs/UserGuide.md

```diff
@@ -182,154 +180,309 @@ Example:
+
+Example:
+* `addpresettask tti/1 tn/Revise for Lab du/3 da/2 t/LSM1301`
+* `addpresettask tti/1 tn/Revise for Lab du/3 da/2`
+
+### Edit Preset Task in Task Template : `editpresettask`
+
+Edit a selected Preset Task in a selected Task Template.
+
+Format: `editpresettask tti/TASKTEMPLATEINDEX pti/PRESETTASKINDEX [tn/TASKNAME] [du/DURATION] [da/DAY] [t/TAG]`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Items in square brackets are optional, but please input at least 1 value to change.
+</div>
+
+- The `TASKTEMPLATEINDEX` must be a **positive integer** and must be **valid**
+- The `PRESETTASKINDEX` must be a **positive integer** and must be **valid**
+- `DAY` have to be an integer within 1-7, with 1 being monday and 7 being sunday
+- `DURATION` will be in hours and less than 24 hours
+- Addition of a preset task to a specific day in the same template will be **rejected** if the total tasks for that day exceed 24 hours
+
+Example: 
+* `editpresettask tti/1 pti/2 tn/CCA lunch du/3 da/5`
+* `editpresettask tti/1 pti/2 tn/CCA lunch du/3 da/5 t/CCA`
+
+### Clear all Preset Task from Task Template : `cleartemplate`
+
+Clear all Preset Task in the selected Task Template.
+
+Format: `cleartemplate tti/TASKTEMPLATEINDEX`
+
+- The `TASKTEMPLATEINDEX` must be a **positive integer** and must be **valid**
+
+Example: 
+* `cleartemplate tti/1 pti/1`
 
-_{explain the feature here}_
+### Delete a Preset Task from Task Template : `deletepresettask`
+
+Remove and Delete a Preset Task from the Task Template.
+
+Format: `deletepresettask tti/TASKTEMPLATEINDEX pti/PRESETTASKINDEX`
+
+- The `TASKTEMPLATEINDEX` must be a **positive integer** and must be **valid**
+- The `PRESETTASKINDEX` must be a **positive integer** and must be **valid**
+
+Example: 
+* `deletepresettask tti/1 pti/1`
+
+### Load Task Template into Task Schedule : `loadtemplate`
+
+Load all the Preset Task listed in Task Template into the Task Schedule.
+
+Format:  `loadtemplate tti/TASKTEMPLATEINDEX`
+
+- The `TASKTEMPLATEINDEX` must be a **positive integer** and must be **valid**
+
+Example:
+*  `loadtemplate tti/1`
+
+### List Preset Task in Task Template : `listpresettask`
+
+List all Preset Task in Task Template.
+
+Format: `listpresettask tti/TASKTEMPLATEINDEX`
+
+- The `TASKTEMPLATEINDEX` must be a **positive integer** and must be **valid**
+
+Example: 
+* `listpresettask tti/1`
+
+### Select Module : `selectmodule` 
+
+Select a Module and load all the ModuleTask into the Module Schedule.
+
+Format: `selectmodule mi/MODULEINDEX`
+
+- The `MODULEINDEX` must be a **positive integer** and must be **valid**
+
+Example: 
+* `selectmodule mi/1`
+
+### Delete Module: `deletemodule`
+
+Remove and Delete a Module and all its Module Task from the Module Schedule.
+
+Format: `deletemodule mi/MODULEINDEX`
+
+- The `MODULEINDEX` must be a **positive integer** and must be **valid**
+
+Example: 
+* `deletemodule mi/1`
+
+### List Module List : `listmodulelist`
```

Comment:
Maybe listallmodules will be a better name

Reply:
Agreed

Reply:
I changeed to `listmodule`